guide/daily-spending-wallet/requesting.md

```diff
@@ -34,439 +34,421 @@ Illustration source
+   alt-text = "Screen showing QR code of a payment address."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
 </div>
 
-### On-chain
+---
 
-Receiving on-chain involves the sender broadcasting a transaction to the network for it to be confirmed.
+## Advanced options
 
-<div class="center" markdown="1">
+When requesting bitcoin there is some options that can offer unique experiences but are more suited for advanced users who know what they are doing.
 
-{% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/confirming.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/confirming@2x.png"
-   layout = "float-left-desktop -background -shadow"
-   caption = "For on-chain, keep users updated on incoming pending transactions."
-   alt-text = "Screen showing on-chain transaction being confirmed."
-   width = 250
-   height = 541
-%}
+### Invoice expirations
 
-As on-chain transaction confirmation times vary based on network congestion and fees being paid, it is uncertain exactly when a on-chain payment will be considered complete. Keep users informed of the state of their incoming transactions.
+Lightning invoices aren't permanent; [they expire over time]({{ "/guide/glossary/#lightning-invoice" | relative_url }}). However, this expiration time can be modified, unlocking unique use cases and improving usability in some situations.
 
-</div>
+When denominating invoices in fiat, custom expiries should be used to prevent exposure to price volatility.
 
-## Success
+If requesting a specific amount denominated in fiat, use a shorter expiry and have the invoice refresh each time it expiries. For a wallet designed for in-person payments, a 30 - 60 second expiry works well.
 
-Wallet applications should clearly indicate once a payment has been received by the user.
+If there is no amount defined on the invoice, the invoice is denominated in bitcoin, or the invoice needs to be shared in a message, then use a longer expiry, like 24 hours.
 
-<div class="center" markdown="1">
+While it is good to allow users to define their own custom expiries, remember that only advanced users will do this. Choose a sensible default that makes sense for your user-base and how the wallet is intended to be used.
+
+For some background on situations the user might encounter with different invoice expiries, see this [blog post](https://d.elor.me/2022/01/lightning-invoice-expiration-ux-considerations/) from designer Stephen DeLorme.
+
+<div class="image-slide-gallery">
 
-{% include image.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/received.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/received@2x.png"
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Metadata.png"
```

Comment:
So I understand, how does the user get to this screen? I feel like I saw 3 share buttons only under the QR code but no "settings" button. They could hit back button to get back to home screen, but I'm not sure where they would go from there.

Reply:
By tapping on the amount / fees copy under the QR code. I will make this more clear. Good catch